be/src/exec/pipeline/aggregate/aggregate_blocking_sink_operator.h

```diff
@@ -54,7 +55,7 @@ class AggregateBlockingSinkOperator : public Operator {
 
 private:
     // Whether prev operator has no output
-    bool _is_finished = false;
+    std::atomic_bool _is_finished = false;
```

Comment:
why change this?

Reply:
some times when BucketSourceOperator call `reset_state` but BucketSinkOperator in other thread will get `_is_finished` is true. I think it's not expected. so I make _is_finished as atomic